hr_maintenance_workspaces/README.rst

```diff
@@ -0,0 +1,21 @@
+=====================
+HR Maintenance Workspaces
+=====================
+
+Allows to associate employees and equipments to the workspaces.
+
+**Table of contents**
+
+.. contents::
+    :local:
+
+Implemented Uses
+================
+
+**Employees**
+
+* Every employee has workspaces associated to it.
+
+**Workspaces**
+
+* Each workspace has employees and equipment asigned to it. It represents a real workspace.
```

Comment:
Use the [README generator](https://github.com/OCA/maintainer-tools#readme-generator) for this

Reply:
Hi, I've tried configurating it and use it, but it didn't generate me anything, could you please generate just a template so I can fill it with the needed information?
Thanks